byterun/win32.c

```diff
@@ -374,9 +387,20 @@ static void expand_pattern(char * pat)
   if (i == 0)
     prefix[0] = 0;
   do {
+#ifdef HAS_WINAPI_UTF16
+    char * aname = utf16_to_utf8(ffblk.name);
+#else
+    char * aname = ffblk.name;
+#endif
     name = caml_stat_strconcat(2, prefix, ffblk.name);
+    free(aname);
```

Comment:
This seems to be another bad rebase: `free(aname)` should be protected by `HAS_WINAPI_UTF16` and the second argument to `caml_stat_strconcat` should also be `aname`.